lib/relistData.ts

```diff
@@ -61,6 +61,6 @@ export const getRelistData = unstable_cache(
   },
   ['full-data'],
   {
-    revalidate: 180,
+    revalidate: 2,
```

Comment:
try not to push this, let's keep it at 180, if you want a lower value on local host we can make it different